code/game/machinery/computer/communications.dm

```diff
@@ -769,8 +769,12 @@
 		if("message")
 			status_signal.data["top_text"] = data1
 			status_signal.data["bottom_text"] = data2
+			log_game("[key_name(usr)] has changed the station status display message to \"[data1] [data2]\" [loc_name(usr)]")
```

Comment:
Why not `key_name_admin`?

Reply:
key name admin shouldn't be used for logging purposes, given it turns it into an anchor to PM the user

Reply:
Ahh I didn't see where this was okay